components/common/src/test/java/com/hotels/styx/config/schema/SchemaTest.java

```diff
@@ -95,10 +83,10 @@ public void bool_validatesBooleanValues() throws Exception {
                 + "  myOkValue2: false \n"
                 + "  myNokValue: x \n");
 
-        bool().validate(ImmutableList.of("myOkValue1"), root, root.get("myOkValue1"), NO_EXTENSIONS);
-        bool().validate(ImmutableList.of("myOkValue2"), root, root.get("myOkValue2"), NO_EXTENSIONS);
+        bool().validate(singletonList("myOkValue1"), root, root.get("myOkValue1"), NO_EXTENSIONS);
+        bool().validate(singletonList("myOkValue2"), root, root.get("myOkValue2"), NO_EXTENSIONS);
```

Comment:
Could we coalesce both `singletonList` and `emptyList` as special cases in our new `listOf` API? That would reduce clutter in the call sites.

Reply:
Explanations again, just FYI. Originally I was trying to use standard Java library methods where possible. But as the `Collections` utilities have grown and become more structured, the more it makes sense to incorporate this kind of logic in them, and use them in preference to standard library methods.